site/quickstart.md

```diff
@@ -120,10 +120,10 @@ kubectl get pods -n sample-weblogic-operator1-ns
 ```
 
 c.	Create an Ingress for the domain, in the domain namespace, by using the [sample](../kubernetes/samples/charts/ingress-per-domain/README.md) Helm chart:
```

Comment:
Using the ingress-per-domain helm chart to create the ingress for the domain suffices, but I find using the host-routing.yaml in kubernetes/samples.charts/traefik/samples gives an idea of how to use the ingress with 2 domains.
kubectl create -f samples/host-routing.yaml
But the way it is right now works as well, I think.